README.md

```
@@ -1,2 +1,81 @@
+```bash
+pnpm db:push
+```
+
+**For Production:**
+
+```bash
+pnpm db:generate
+pnpm db:migrate
+```
+
+**To run Drizzle Studio**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)